### PR TITLE
fix: template version mission subpath

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**

template version missing subpath.

**Solution:**

add subpath for template version when import a template with subpath

**Related Issue:**

#1579

### CODE REVIEW

File: pkg/templates/git.go [hunk 1]
```diff
@@ -345,6 +350,7 @@ func GetTemplateVersions(
 	entity *model.Template,
 	newVersions []*version.Version,
 	versionSchema map[*version.Version]types.TemplateVersionSchema,
+	subPath string,
```
Comment (gitlawr) [inline pkg/templates/git.go:353]: Is subPath included in `entity.Source`?

File: pkg/templates/git.go [hunk 2]
```diff
@@ -33,6 +33,7 @@ func CreateTemplateVersionsFromRepo(
 	entity *model.Template,
 	versions []*version.Version,
 	versionSchema map[*version.Version]types.TemplateVersionSchema,
+	subPath string,
```
Comment (gitlawr) [inline pkg/templates/git.go:36]: Is subPath included in entity.Source?

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by thxCode — APPROVED** [review]
LGTM

**Pull request merged**